src/main/java/com/khutircraftubackend/jwt/exception/JwtExceptionHandler.java

```diff
@@ -0,0 +1,24 @@
+package com.khutircraftubackend.jwt.exception;
+
+import com.auth0.jwt.exceptions.JWTVerificationException;
+import com.auth0.jwt.exceptions.TokenExpiredException;
+import lombok.extern.slf4j.Slf4j;
+import org.springframework.http.HttpStatus;
+import org.springframework.web.bind.annotation.ControllerAdvice;
+import org.springframework.web.bind.annotation.ExceptionHandler;
+import org.springframework.web.bind.annotation.ResponseStatus;
+
+@ControllerAdvice
+@Slf4j
+@ResponseStatus(HttpStatus.UNAUTHORIZED)
```

Comment:
probably wrong place for this annotation.
i guess it should be applied on the method below:
```java
@ResponseStatus(HttpStatus.UNAUTHORIZED)
public String handleJWTException(RuntimeException e) {
  ....
}
```

please check indents as well.